lua/telekasten.lua

```diff
@@ -251,16 +251,15 @@ local function imgFromClipboard()
     -- 00000090  10 66 d7 01 b1 e4 fb 79  7c f2 2c e7 cc 39 e7 3d  |.f.....y|.,..9.=|
 
     local pngname = "pasted_img_" .. os.date("%Y%m%d%H%M%S") .. ".png"
-    local pngpath = M.Cfg.home
+    local pngpath = M.Cfg.home .. "/" .. pngname
     local relpath = pngname
 
     if M.Cfg.image_subdir then
         relpath = Path:new(M.Cfg.image_subdir):make_relative(M.Cfg.home)
             .. "/"
             .. pngname
-        --pngpath = M.Cfg.home .. "/" .. M.Cfg.image_subdir
+        pngpath = M.Cfg.image_subdir .. "/" .. pngname
     end
```

Comment:
While I am at it, I think the link could be improved a bit. Linking with `relpath` as it is done now will probably look quite messy if `image_subdir` is not an actual subdir of `zk home`. I would maybe be nicer if we format the link as:
1. Relative in case if 'image_subdir` is a subdir of `zk home` 
2. Absolute if `image_subdir` is outside `zk home`

Reply:
OK - my use case should remain unaffected by that :smile:, so I don't object.

Reply:
Perfect, I will add that as well then.

Reply:
Thanks, this is awesome!

Reply:
Actually I did some testing, and it seems that this is already taken care of by `Path:make_relative`...

I was afraid that we would end up with links like `[../../../../user/myimage_dir]`but no, if the dir is not a subdir of ZK, it will use its absolute path. So all good here